<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebEnginePdf</class>
 <widget class="QMainWindow" name="WebEnginePdf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebEnginePdf</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabHtmlSource">
    <attribute name="title">
     <string>HTML Sources</string>
    </attribute>
    <widget class="QTextEdit" name="edHtmlSource">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>371</width>
       <height>361</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&amp;lt;!DOCTYPE html&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&amp;lt;html lang=&amp;quot;en&amp;quot;&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    &amp;lt;head&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        &amp;lt;title&amp;gt;Sample Page&amp;lt;/title&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        &amp;lt;style&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        body {&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        font-family: Arial, Helvetica, sans-serif;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        }&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        @media print {&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        .pagebreak { page-break-before: always; }&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        }&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        @page {&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        size: A5 portrait;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        }        &lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        &amp;lt;/style&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    &amp;lt;/head&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    &amp;lt;body&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        &amp;lt;h1&amp;gt;Head 1&amp;lt;/h1&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        &amp;lt;h2&amp;gt;Head 2&amp;lt;/h2&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        &amp;lt;div&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;            &amp;lt;img src=&amp;quot;data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAUAAAAFCAYAAACNbyblAAAAHElEQVQI12P4//8/w38GIAXDIBKE0DHxgljNBAAO9TXL0Y4OHwAAAABJRU5ErkJggg==&amp;quot; alt=&amp;quot;Red dot&amp;quot; /&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;            &amp;lt;/div&amp;gt;        &amp;lt;p&amp;gt; Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam lorem erat, lobortis sit amet facilisis eu, maximus eget sem. In consectetur tincidunt nisi sit amet molestie. Phasellus nibh risus, egestas quis vestibulum ut, blandit vel nisi. Proin lobortis risus sed dignissim mollis. Etiam nec varius erat, pharetra cursus dui. Pellentesque fermentum nec lacus sed eleifend. Nunc hendrerit semper dignissim. Sed venenatis, turpis quis pharetra luctus, felis nisi feugiat justo, at accumsan sem dui ac purus. Vestibulum lectus purus, consectetur et urna tincidunt, maximus cursus felis. Aliquam tincidunt commodo diam id rutrum. Sed gravida sit amet orci vel venenatis. Nunc congue nibh eu sem tempus, sit amet fermentum nisi tincidunt. Integer eu interdum enim, sed scelerisque nisi. Donec accumsan lacus libero, sit amet dictum neque consequat feugiat. Vestibulum aliquet mi eu malesuada tincidunt.&amp;lt;/p&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;            &amp;lt;h1 class=&amp;quot;pagebreak&amp;quot;&amp;gt;Head 3&amp;lt;/h1&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;            &amp;lt;h2&amp;gt;Head 4&amp;lt;/h2&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;            &amp;lt;p&amp;gt; Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam lorem erat, lobortis sit amet facilisis eu, maximus eget sem. In consectetur tincidunt nisi sit amet molestie. Phasellus nibh risus, egestas quis vestibulum ut, blandit vel nisi. Proin lobortis risus sed dignissim mollis. Etiam nec varius erat, pharetra cursus dui. Pellentesque fermentum nec lacus sed eleifend. Nunc hendrerit semper dignissim. Sed venenatis, turpis quis pharetra luctus, felis nisi feugiat justo, at accumsan sem dui ac purus. Vestibulum lectus purus, consectetur et urna tincidunt, maximus cursus felis. Aliquam tincidunt commodo diam id rutrum. Sed gravida sit amet orci vel venenatis. Nunc congue nibh eu sem tempus, sit amet fermentum nisi tincidunt. Integer eu interdum enim, sed scelerisque nisi. Donec accumsan lacus libero, sit amet dictum neque consequat feugiat. Vestibulum aliquet mi eu malesuada tincidunt.&amp;lt;/p&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;        &amp;lt;/body&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;    &amp;lt;/html&amp;gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnToBrowser">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>To Browser -&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWebWidget">
    <attribute name="title">
     <string>Browser View</string>
    </attribute>
    <widget class="QWebEngineView" name="webView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>371</width>
       <height>361</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSavePDF">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Save To PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnLoadPDF">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Load PDF -&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabPdfWidget">
    <attribute name="title">
     <string>PDF View</string>
    </attribute>
    <widget class="QPushButton" name="btnPrev">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNext">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbPages">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Page X / Y</string>
     </property>
    </widget>
    <widget class="QPdfView" name="pdfView" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>371</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebEngineView</class>
   <extends>QWidget</extends>
   <header location="global">QtWebEngineWidgets/QWebEngineView</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QPdfView</class>
   <extends>QWidget</extends>
   <header location="global">QtPdfWidgets/qpdfview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
